rasa/core/training/dsl.py

```diff
@@ -181,24 +181,55 @@ def __init__(self, domain, interpreter, template_vars=None, use_e2e=False):
             story_steps.extend(steps)
 
         # if exclusion percentage is not 100
-        if exclusion_percentage and exclusion_percentage is not 100:
+        if exclusion_percentage and exclusion_percentage != 100:
             import random
             idx = int(round(exclusion_percentage / 100.0 * len(story_steps)))
             random.shuffle(story_steps)
             story_steps = story_steps[:-idx]
 
         return story_steps
 
+    def verify_stories_format(self, filename):
+        stories = self._remove_comments(filename)
```

Comment:
Why do you need to remove the comments to check the story lines?

Reply:
So when I check if the lines starts with #, - or * I'm sure it is not a comment, because if it is, it doesn't have to start with these symbols.